src/features/organizations/components/ActivistPortlHeader/index.tsx

```diff
@@ -27,8 +30,27 @@ const ActivistPortalHeader: FC<Props> = ({
   topLeftComponent,
 }) => {
   const user = useUser();
+  const messages = useMessages(messageIds);
 
   const hasNavBar = selectedTab && tabs;
+  const [logoutMenuAnchorEl, setLogoutMenuAnchorEl] =
+    useState<HTMLButtonElement | null>(null);
+  const router = useRouter();
+  const path = usePathname();
+
+  const menuItems: MenuItem[] = [
+    {
+      label: messages.home.menu.logout(),
+      onClick: () => router.push('/logout'),
+    },
+  ];
+  if (path !== null && !path.startsWith('/my')) {
```

Comment:
If for some reason `path` becomes `undefined`, I believe that this line of code would crash. The strict equality with `null` will be false, and the second expression will evaluate, trying to access `startsWith` on `undefined`.

For this reason, I avoid strict equality. I have elaborated in another PR why I have a gut reaction to strict equality in general in typescript. I'm dropping a link here in case you're curious.
https://github.com/zetkin/app.zetkin.org/pull/2426#discussion_r1887956255

I'm curious to hear your thoughts on this!